.github/workflows/tag.yml

```diff
@@ -0,0 +1,43 @@
+name: Set git tag
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+    inputs:
+      pushDateTag:
+        description: "Push datestamp (production) tag"
+        type: boolean
+        required: false
```

Comment:
We no longer need the `inputs` section here, dispatch can just implicitly mean that a tag should be pushed (as that's all this workflow does)